packages/react/src/atomic/Icon/src/demo/Render/render.stories.tsx

```
@@ -0,0 +1,73 @@
+        role="list"
+        aria-label="Icon showcase"
+      >
+        {Icons.map(
+          ([name, IconComponent]: [string, React.ComponentType<any>]) => (
+            <Box
+              key={name}
+              margin="3"
+              padding="1"
+              borderStyle="dashed"
+              borderColor="danger-surface"
+              role="listitem"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<li>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)